linode/linode_client.py

```diff
@@ -477,22 +483,26 @@ def _api_call(self, endpoint, model=None, method=None, data=None, filters=None):
 
         body = json.dumps(data)
 
-        r = method(url, headers=headers, data=body)
+        response = method(url, headers=headers, data=body)
+
+        warning = response.headers.get('Warning', None)
```

Comment:
is it possible other endpoints/classes may receive warning headers in the future? perhaps this logic should be abstracted out from get_distributions? just a thought

Reply:
This is the logic for all api requests, so he did it just as you suggested.